jina/serve/stream/__init__.py

```diff
@@ -62,24 +64,38 @@ def __init__(
         self.total_num_floating_tasks_alive = 0
 
     async def stream(
-        self, request_iterator, context=None, results_in_order: bool = False, *args
+        self,
+        request_iterator,
+        context=None,
+        results_in_order: bool = False,
+        prefetch: Optional[int] = None,
+        *args,
     ) -> AsyncIterator['Request']:
         """
         stream requests from client iterator and stream responses back.
 
         :param request_iterator: iterator of requests
         :param context: context of the grpc call
         :param results_in_order: return the results in the same order as the request_iterator
+        :param prefetch: How many Requests are processed from the Client at the same time. If not provided then the prefetch value from the metadata will be utilized.
         :param args: positional arguments
         :yield: responses from Executors
         """
+        prefetch = self._prefetch
```

Comment:
this is weird, it is either an input parameter or read from a private method, but seems weird this usage